src/nat/llm/huggingface_llm.py

```
@@ -0,0 +1,402 @@
+    _torch: Any
+
+    def __init__(self, model_name: str, config: HuggingFaceConfig):
+        """Initialize HuggingFace model wrapper.
+
+        Args:
+            model_name: Name of the loaded model
+            config: Configuration for the model
+        """
+        # Get from cache
+        if model_name not in _model_cache:
+            raise ValueError(f"Model {model_name} not loaded in cache")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)